src/nationalcareers_toolkit/assets/src/frontend/js/compui/apps/dfc-app-findacourse.js

```diff
@@ -94,14 +102,6 @@ $(document).ready(function () {
                     $('.find-a-course-page #distance-block').hide();
                     $(".find-a-course-page #orderBy-Input option[value='Distance']").remove();
                 }
-                $('#fac-result-list').html("");
-                $('#fac-result-list').html(parsedData.html);
-                $('.fac-result-count').html("");
-                $('.fac-result-count').html(addCommas(parsedData.count));
-                $("#fac-clear-filters").show();
-                $(".fac-filters-block").show();
-                let searchTerm = $('.find-a-course-page #search-input').val();
-                $(".fac-filters-block").html("<p id='fac-clear-filters'><a href='/find-a-course/searchcourse?searchTerm=" + searchTerm + "' aria-label='ClearFilters'>Clear filters</a></p>");
             },
             failure: function (jqXHR, textStatus, errorThrown) {
```

Comment:
I think we should avoid the Alert here. We should be displaying and on screen error or redirecting to a service down/ error page.